python/boat_controller_node.py

```diff
@@ -31,7 +32,7 @@ def windSensorCallBack(windsensor_msg_instance):
 
     global apparentWindAngleRad
 
-    apparentWindAngleDegrees = windsensor_msg_instance.measuredDirectionDegrees
+    apparentWindAngleDegrees = windsensor_msg_instance.measuredBearingDegrees
```

Comment:
Yes, see my conversion notes linked in the issue description

Reply:
Oh I see. As far as I understand it, these are the conventions used by pathfinding, correct? As long as we follow then we should be fine.

Reply:
Assuming you follow the same conventions as the raw sensor data (0 is north/forward, 90 is east/right - cw) then yes

Reply:
I believe that as long as the desired heading that we get from pathfinding matches the current heading reading's convention, it should be fine. Can you confirm this?

Reply:
If you mean that they follow the same conventions as `gps_can_true_heading_degrees` and `wind_sensor_1_angle_degrees` from the `/sensors` topic which you used previously, then yes I can

Reply:
Kinda. What I am more concerned about is that if `/bearingDegrees` matches the conventions of `/desired_heading_degrees` from pathfinding. If not, then our computations for the rudder will be wrong. Furthermore, the convention for `/measuredBearingDegrees` for the wind angle should also be the same as the latter two. Otherwise, our calculations for the sail will also be wrong.

Reply:
Let's discuss this in tomorrow's meeting